### DISCUSSION TITLE
Is there support for pip --constraint options?

### DISCUSSION THREAD

**Discussion by songololo** [pdm-project/pdm#1352] [Q&A]
Is there a way to pass a `--constraint` option to `pdm`?

For example, `airflow` requires installation via `pip` and a constraints file, e.g.
`pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"`

Is there a way to run this from `pyproject.toml` via `pdm`?

**Comment by frostming** [comment]
No, there is no equivalent in PDM
